megatron/model/norms.py

```diff
@@ -13,8 +13,8 @@
 # limitations under the License.
 
 import torch
-from torch.nn import LayerNorm as LayerNorm
-
+# from torch.nn import LayerNorm as LayerNorm
```

Comment:
We need to add a config arg for the fused layernorm, and turn it on by default. People may want to turn it off. We should treat the layernorm kernel the same as our other fused kernels: https://github.com/EleutherAI/gpt-neox/blob/98716ebb4818498fe118ddb07c62bfa94a48da2a/megatron/neox_arguments/neox_args.py#L243-L261